azure-communication-ui/azure-communication-ui/src/main/res/layout/azure_communication_ui_bottom_drawer_title_cell.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+   Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
first line, above the XML even

Reply:
The xml version declaration _must_ be the first line of the file as per the spec. Android Studio gives me an error if I attempt moving the copyright to the top: `Xml declaration should precede all document content`